modules/media/jsx/mediaIndex.js

```diff
@@ -56,6 +74,7 @@ class MediaIndex extends Component {
    * @return {*} a formated table cell for a given column
    */
   formatColumn(column, cell, row) {
+    cell = this.mapContainerColumns(column, cell);
```

Comment:
Media module not loading. This line is giving an error

![LORIS#5291-consoleError](https://user-images.githubusercontent.com/34260251/68874971-0ed49b00-06d0-11ea-8607-17365a9dc0ca.png)